src/tests/cli/test_logout.py

```diff
@@ -1,15 +1,26 @@
 # from contextlib import redirect_stdout, redirect_stderr
 # from io import StringIO
 import unittest
+import os
 
 from multiprocessing import get_start_method
 from unittest.mock import call
 
 from ..base import IntegrationTests
 
 
+@unittest.skipIf(
+    os.environ.get('AWSCLI_LOGIN_FAST_TEST_ONLY'), 'Skipping slow test')
 class TestNoProfile(IntegrationTests):
-    """ Integration tests for no profile. """
+    """ Integration tests for no profile.
+
+    This is our slowest test - at 2.16 seconds,
+    and so runs only in the testenv:slow tox environment,
+    which overrides the .skip above.
+
+    Our next slowest test costs .06 seconds, as of Aug 2023.
+    Tested with `pytest --durations=10`
+    """
```

Comment:
I added this comment to facilitate discussion. It can probably be trimmed down or removed.

I'm thinking the expected time cost of ~2 seconds is valuable, but that might be all we keep?